examples/CMakeLists.txt

```diff
@@ -60,7 +60,7 @@ dd4hep_enable_tests (
   DDG4_MySensDet
   DDCodex
   )
-if ( "${ROOT_VERSION}" VERSION_GREATER "6.17.00" )
+if ( "${ROOT_VERSION}" VERSION_GREATER "6.20.00" )
```

Comment:
```suggestion
if ( "${ROOT_VERSION}" VERSION_GREATER "6.17.99" )
```

Reply:
Changed to `if ( "${ROOT_VERSION}" VERSION_GREATER_EQUAL "6.20.00" )` which is
what is required in `OpticalSurfaces/CMakeLists.txt`

Reply:
There is just one test in OpticalSurfaces which requires root 6.20, the others require 6.18.

Reply:
What it needs is ROOT 6.18.
The GDML example anyhow explicitly requires 6.20.

Reply:
Yes, but if you then have ROOT_VERS<6.20 the cmake crashes - so that's not what you want. I guess the 'OpticalSurfaces/CMakeLists.txt' should be changed accordingly (try to build the GDML stuff only if ROOT_VERS>=6.20).

Reply:
Ok, I realize that Markus had fixed it in the meantime - so reverted back to 
` "${ROOT_VERSION}" VERSION_GREATER_EQUAL "6.18.00"`.